data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,465 @@
+			`	Do you want to investigate?`
+			choice
+				`	(Investigate.)`
+				`	(Come back later.)`
+					defer
+			`	You knock on the steel sheet, and a woman within the alcove slides it to the side. As she emerges, she carefully moves her body to block your view of the alcove.`
+			`	She asks nervously, "Yes, can I help you?"`
+			choice
+				`	"Why are you hiding a Hai in there?"`
+					goto hiding
+				`	"Can I help you?"`
+					goto help
+				`	"No, I was just walking by. Sorry to bother you."`
+					decline
+			label hiding
+			`	"Please don't tell anyone! We were both kidnapped, but we managed to escape. Now we're hiding here, hoping the men don't find us again.`
+			`	"Actually, if you can give us a ride to <planet>, we can pay when we get there. You'll be followed though, so I hope you have a fast ship."`
+				goto "come in"
+			label help
+			`	"Yes! Some men kidnapped my friend and I, but she got us out and we managed to escape. We've been hiding here, hoping they don't find us again. If you can fly us to <planet>, we can pay you when we arrive. I hope you have a fast ship; they'll probably chase us."`
+				goto "come in"
+			label "come in"
+			`	"Come in, and we'll talk about it."`
+			choice
+				`	(Enter their alcove.)`
+				`	"No, I can't help you get home."`
+					decline
+			`	The alcove is a dead air space between two utility rooms in the basement of a warehouse. The hole in the wall looks like it was recently dug out by a laser cutter. There's a small refrigerator, a two-person cot, an air processor, and a shelf with typical household items. Books are strewn everywhere; the women must be avid readers. It seems they've set up this tiny space as a decent, albeit small, apartment.`
+			`	The human woman slides the sheet of steel back into place behind you both once you're in and says, "I'm Lucy."`
+			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a taser. I woke up in the brig of a space ship."`
+			`	"My experience was similar. I was walking from my biolab to my house on Serpens. Some men jumped out from behind a dune and zapped me."`
+			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks. Please take us to <destination>!"`
+			choice
+				`	"Sure. Let's go."`
+				`	"No, you'll have to find another pilot."`
+					decline
+			`	The two women put on clothing that thoroughly covers their person. They lead you through a series of ducts and alleys back to the building where your ship is parked. When you have fifty meters left to go, someone shoots Meroot with a stun gun. You and Lucy manage to carry her to your ship and lift off. It appears you're being followed into orbit.`
```

Comment:
In that distance, neither I nor Lucy are shot? Even though we're now moving slower than before? The pursuers don't even catch up?